config/hwloc.m4

```diff
@@ -1019,6 +1019,39 @@ return clGetDeviceIDs(0, 0, 0, NULL, NULL);
     fi
     # don't add LIBS/CFLAGS/REQUIRES yet, depends on plugins
 
+    # RSMI support
+    hwloc_rsmi_happy=no
+    if test "x$enable_io" != xno && test "x$enable_rsmi" != "xno"; then
+      hwloc_rsmi_happy=yes
+      AC_CHECK_HEADERS([rocm_smi.h], [
+        AC_CHECK_LIB([rocm_smi64], [rsmi_init], [HWLOC_RSMI_LIBS="-lrocm_smi64"], [hwloc_rsmi_happy=no])
+        ], [hwloc_rsmi_happy=no])
```

Comment:
Is there a librocm_smi32.so for 32bits systems?
Add a comment so that we know why we're only looking for the 64bits lib without checking if we're on a 64bit machine.

Reply:
There is no  librocm_smi32.so, they call library  librocm_smi64.so

Reply:
Indeed the CMakeLists.txt only enables 64bits by default, and manual changes to switch to 32bits don't touch the library name. Please put a comment about this :)

Reply:
will add a conmments.